src/lbaf/Model/lbsObject.py

```diff
@@ -83,11 +110,27 @@ def __init__(
             raise TypeError(f"subphases: {subphases} is of type {type(subphases)} but must be <class 'list'>")
 
     def __repr__(self):
-        return f"Object id: {self.__index}, load: {self.__load}"
+        return f"Object id: {self.__seq_id}, load: {self.__load}"
```

Comment:
Could this use the `get_id()` function you wrote (that handles either type of id)?

```python
def __repr__(self):
    return f"Object ID: {get_id()}, load: {self.__load}"
```

Reply:
This is done